### DISCUSSION TITLE
Actively learn revisions

### DISCUSSION THREAD

**Discussion by Aggeloskourtesis** [courses-ionio/hci#1635] [Announcements]
Καλησπέρα, παρατήρησα στις ερωτήσεις κειμένου το κουμπί "save draft". Μπορώ να το χρησιμοποιήσω για να προχωρίσω στην επόμενη ερώτηση και μετα να γυρίσω για να κάνω "Submit" ;

**Comment by marioskomianos** [comment]
Λογικα κανωντας save draft συνεχιζεις τις ερωτησεις και επιστρεφεις οταν θες και κανεις submit.
